src/config/config.ts

```diff
@@ -0,0 +1,25 @@
+import { workspace } from "vscode";
+
+export type CustomWrapConfig = {
+  name: string,
+  id: string,
+  body: Array<string>,
+};
+
+export class FlutterPlusConfig {
+  private static instance: FlutterPlusConfig;
+
+  public static getInstance(): FlutterPlusConfig {
+    if (!FlutterPlusConfig.instance) {
+      FlutterPlusConfig.instance = new FlutterPlusConfig();
+    }
+    return FlutterPlusConfig.instance;
+  }
+
+  public getCustomWraps(): Array<CustomWrapConfig> {
+    const config = workspace.getConfiguration('flutterplus');
```

Comment:
Please check that the current `workspace` provides configurations for workspace settings and global settings.

Replace `flutterplus` with `flutter-plus`